inst/example_app/app.R

```diff
@@ -283,7 +283,6 @@ shiny::shinyApp(
               ),
               heading_text("govTabs", size = "s"),
               shinyGovstyle::govTabs("tabsID", data, "tabs"),
-              heading_text("button_Input", size = "s", id = "button_input_tables_tabs_accordions"),
```

Comment:
One very nitpicky thing, would we add a bit more space between the table and the accordions heading? 
![image](https://github.com/user-attachments/assets/34cf9edb-62dc-45d1-a593-f88aa8a49b12)